be/src/exprs/vectorized/array_functions.cpp

```diff
@@ -959,6 +959,10 @@ ColumnPtr ArrayFunctions::array_contains_all([[maybe_unused]] FunctionContext* c
     return ArrayHasImpl<false>::evaluate(*arg0, *arg1);
 }
 
+ColumnPtr ArrayFunctions::array_map([[maybe_unused]] FunctionContext* context, const Columns& columns) {
```

Comment:
add some comment here to explain why return columns[0]

Reply:
ok, change it to return nullptr with comments.